### PR TITLE
Adds release notes 2.5

### PR DESCRIPTION
**Pull request by kolchfa-aws**
Adds release notes 2.5

### Checklist
- [x ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: release-notes/opensearch-documentation-release-notes-2.5.0.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# OpenSearch Documentation Website 2.5.0 Release Notes
+
+The following page provides a list of updates made to OpenSearch documentation for OpenSearch 2.5.0.
+
+## New documentation for 2.5.0
+
+- Add Query String for rollups [#2428](https://github.com/opensearch-project/documentation-website/pull/2428)
+- Updates to Security Analytics documentation [#2408](https://github.com/opensearch-project/documentation-website/pull/2408)
+- Add new Rules documentation that covers YAML Editor view [#2407](https://github.com/opensearch-project/documentation-website/pull/2407)
+- Adds documentation for Admin UI index operations [#2403](https://github.com/opensearch-project/documentation-website/pull/2403)
```
Comment (ariamarble) [inline release-notes/opensearch-documentation-release-notes-2.5.0.md:10]: Should we rename this to match our changes? or are we sticking with the PR name?

**Review by ariamarble — APPROVED** [review]
LGTM - left a comment for naming

File: release-notes/opensearch-documentation-release-notes-2.5.0.md (hunk 1 above)
Comment (kolchfa-aws) [reply release-notes/opensearch-documentation-release-notes-2.5.0.md:10]: @ariamarble Do you have the ability to still change the PR name? If not, I think it's fine to leave it as is.

**Review by vagimeli — APPROVED** [review]
LGTM

**Pull request merged**